src/traces/image/defaults.js

```diff
@@ -11,13 +11,19 @@
 var Lib = require('../../lib');
 var attributes = require('./attributes');
 var constants = require('./constants');
+var dataUri = require('../../snapshot/helpers').IMAGE_URL_PREFIX;
 
 module.exports = function supplyDefaults(traceIn, traceOut) {
     function coerce(attr, dflt) {
         return Lib.coerce(traceIn, traceOut, attributes, attr, dflt);
     }
+    coerce('source');
+    if(traceOut.source && !traceOut.source.match(dataUri)) traceOut.source = null;
```

Comment:
Please let me know what's the best way to handle this: basically, we don't want a `source` attribute that isn't a data URIs for an image. Is setting it to `null` OK?

Reply:
Since `source` is of type `string` I think we could add a blank `dflt: ''` to source attribute.
Then here set it to `attributes.source.dflt` or alternatively just
```js
delete traceOut.source
```

Reply:
I went with `delete traceOut.source` in e6934467faf4ca8a5e2a83ef4339ab9f8fd68dc7!

Thanks @archmoj